Core/main.cpp

```diff
@@ -44,6 +51,54 @@ int main(int argc, char* argv[])
 	}
 
 	glfwTerminate();
+}
+
+
+
+void run_physics(Physics& physx) {
+	while (true) {
+		physx.Update();
+		std::this_thread::sleep_for(std::chrono::milliseconds(1000));
+	}
+}
+
+void run_ui(UI& ui) {
+	while (true) {
+		ui.Update();
+		std::this_thread::sleep_for(std::chrono::milliseconds(1000));
+	}
+}
+
+
+
+int main(int argc, char* argv[])
+{
+
+	std::cout << "Do you know what DK Stands for? Donkey Kong? Nah. Drift King." << std::endl;
+
+
+	// TODO: Make this a call to the Render Project
+	//run_glfw();
+
+	UI ui;
+	Physics physx;
+	//Render render;
+
+	ui.initialize();
+	physx.initialize();
+	//render.initialize();
+
+
+	//std::thread render_thread(run_glfw);
+	std::thread physics_thread(run_physics, std::ref(physx));
+	std::thread ui_thread(run_ui, std::ref(ui));
```

Comment:
If this was placeholder for now we should restructure it such that code going into develop actually functions.